backend/tag/permissions.py

```diff
@@ -0,0 +1,6 @@
+from rest_framework.permissions import BasePermission
+
+
+class IsTagOwner(BasePermission):
+    def has_object_permission(self, request, view, obj):
+        return request.user.is_superuser or (obj.user == request.user)
```

Comment:
I'm not 100% sure we want to give superusers ownership over all tags. In some projects (People & Parliament, for instance), researchers may be superusers, who may be interested in the tagging feature, but sandboxed in their own tagging system - or does this permission only apply for the admin interface? In there, it may be certainly a nice feature to be able to supervise tags of other users.

Reply:
From what I can tell, this does not affect the queryset in the TagViewset below (which filters on `user=self.request.user`).

That said, this sounds like maybe researchers should not be superusers, and instead be staff users with permissions for specific admin tasks.

Reply:
From what I can tell, this does not affect the queryset in the TagViewset below (which filters on `user=self.request.user`).

That said, this sounds like maybe researchers should not be superusers, and instead be staff users with permissions for specific admin tasks.